file-service-persistence/src/main/java/uk/gov/justice/services/fileservice/client/FileService.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.justice.services.fileservice.client;
+
+
+import uk.gov.justice.services.fileservice.api.FileRetriever;
+import uk.gov.justice.services.fileservice.api.FileStorer;
+import uk.gov.justice.services.fileservice.domain.FileReference;
+import uk.gov.justice.services.fileservice.repository.FileStore;
+
+import java.io.InputStream;
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.inject.Inject;
+import javax.json.JsonObject;
+
+/**
+ * Implementation of the {@link FileStorer} and {@link FileRetriever}
+ * interfaces, as the entry point into the file service.
+ *
+ * Currently this class is a simple wrapper around {@link FileStore}, kept
+ * separate simply for isolation of responsibilities
+ */
+public class FileService implements FileStorer, FileRetriever {
+
+    @Inject
+    FileStore fileStore;
+
+    /**
+     * Stores a new file in the database.
+     *
+     * @param metadata The json metadata of the file to be stored as a {@link JsonObject}.
+     * @param fileContentStream The {@link InputStream} of the file to be stored
+     */
```

Comment:
No need to have duplicated Javadoc on @Override methods - just make sure the methods on the interfaces being implemented have these javadocs in full.